subsys/logging/Kconfig

```diff
@@ -86,7 +86,7 @@ config LOG_MIPI_SYST_ENABLE
 
 if !LOG_MINIMAL
 
-menu "Prepend log message with function name"
+menu "Prepend standard log message with function name"
```

Comment:
Maybe "non-hexdump" instead of "standard" as I don't know if the term "standard" is defined anywhere in the kconfig tree? WE

Reply:
done